File: basic/src/main/java/com/yuanshenbin/basic/model/PreviewImgModel.java
package com.yuanshenbin.basic.model;

import java.io.Serializable;
import java.util.List;

/**
 * author : yuanshenbin
 * time   : 2021-11-08
 * desc   : 预览大图
 */
public class PreviewImgModel implements Serializable {
    private List<String> path;
    private int index;

    public List<String> getPath() {
        return path;
    }

    public void setPath(List<String> path) {
        this.path = path;
    }

    public int getIndex() {
        return index;
    }

    public void setIndex(int index) {
        this.index = index;
    }
}

File: basic-kotlin/src/main/java/com/yuanshenbin/basic/model/PreviewImgModel.kt
package com.yuanshenbin.basic.model

import android.content.res.Configuration
import java.io.Serializable

/**
 * author : yuanshenbin
 * time   : 2021-11-08
 * desc   : 预览大图
 */
class PreviewImgModel : Serializable {
    var path: List<String>? = null
    var index = 0
    var orientation = Configuration.ORIENTATION_PORTRAIT

}